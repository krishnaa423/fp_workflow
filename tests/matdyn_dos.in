&INPUT
asr='crystal'
flfrc='struct.fc'
flfrq='struct.phdos.freq'
flvec='struct.phdos.modes'
dos=.true.
fldos='struct.phdos'
nk1=2
nk2=2
nk3=2
/
