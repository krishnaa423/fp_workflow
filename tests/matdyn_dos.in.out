
     Program MATDYN v.7.2 starts on 17Jul2024 at  5:10:20 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
         "P. Giannozzi et al., J. Phys.:Condens. Matter 29 465901 (2017);
         "P. Giannozzi et al., J. Chem. Phys. 152 154105 (2020);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote

     Parallel version (MPI), running on     1 processors

     MPI processes distributed on     1 nodes
     28142 MiB available memory on the printing compute node when the environment starts
 
 mass for atomic type   1 not given; uses mass from file struct.fc                                                                                                                                                                                                                                                       
 Norm of the difference between old and new effective charges:    0.00000000000000000000

 Norm of the difference between old and new force-constants:   0.00018535288883640358

 
     MATDYN       :      0.03s CPU      0.03s WALL

 
   This run was terminated on:   5:10:20  17Jul2024            

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
Warning: ieee_inexact is signaling
FORTRAN STOP
